apps/fiona/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ add_executable (fiona_illumina fiona.cpp)
 
 set_target_properties (fiona_illumina PROPERTIES COMPILE_DEFINITIONS "FIONA_ILLUMINA")
 
+# Boost 1.74.0 is not compatible with cpp20
```

Comment:
Could we upgrade to a newer version later?

Reply:
We could, but there is no compatible version yet.

Reply:
Ah okay, so that might be a move in the future :)